styles/ShopUp.css

```diff
@@ -125,6 +144,33 @@
     width: 25rem;
     height: auto;
     overflow: hidden;
+    position: relative;
+}
+
+.product-icon {
+    font-size: 1.8rem;
+    padding: 0.6rem;
```

Comment:
you can use px for values less than 10px

Reply:
Sure I'll look into that Thanks for reviewing